src/meshapi/urls.py

```diff
@@ -30,6 +30,12 @@
     path("sectors/<uuid:pk>/", views.SectorDetail.as_view(), name="meshapi-v1-sector-detail"),
     path("accesspoints/", views.AccessPointList.as_view(), name="meshapi-v1-accesspoint-list"),
     path("accesspoints/<uuid:pk>/", views.AccessPointDetail.as_view(), name="meshapi-v1-accesspoint-detail"),
+    path("billing/installfeedata/", views.InstallFeeBillingDataList.as_view(), name="meshapi-v1-installfeedata-list"),
+    path(
+        "billing/installfeedata/<uuid:pk>/",
```

Comment:
should we slug this?
```suggestion
        "billing/install-fee-data/<uuid:pk>/",
```

Reply:
Is this our first multi-word path? I don’t care either way as long as we’re consistent

Reply:
i don't feel terribly strongly about it

Reply:
We're doing kebab case in other places. Good catch